source/MaterialXTest/MaterialXRuntime/Runtime.cpp

```diff
@@ -2296,6 +2297,63 @@ TEST_CASE("Runtime: Commands", "[runtime]")
     mx::RtCommandResult unknownResult;
     mx::RtCommand::createPrim(stage, mx::RtToken("unknown"), mx::RtPath("/"), mx::EMPTY_TOKEN, unknownResult);
     REQUIRE(!unknownResult);
+
+	//
```

Comment:
Test case for the new commands.

Reply:
Would be good to add observation testing as well as well as failure case testing.